photon-ml/src/main/scala/com/linkedin/photon/ml/avro/model/ModelProcessingUtils.scala

```diff
@@ -109,7 +118,8 @@ object ModelProcessingUtils {
         val Array(featureShardId) = IOUtils.readStringsFromHDFS(idInfoPath, configuration).toArray
 
         // Load the coefficients
-        val featureNameAndTermToIndexMap = featureShardIdToIndexMapLoader(featureShardId).indexMapForDriver()
+        val indexMapLoaders = featureShardIdToIndexMapLoader.flatMap(_.get(featureShardId))
+        val featureNameAndTermToIndexMap = indexMapLoaders.map(_.indexMapForDriver())
```

Comment:
It's pretty inconsistent in the Spark code. In some places, they prefix the var with "opt", in others they don't specify at all. I lean more toward not specifying, because we don't include var types in the names in any other case, and like @ferd36 said, it makes the var names very long.